translation/pt/patterns/core-team.md

```diff
@@ -0,0 +1,120 @@
+  * Bugs frequentes.
+  * Configuração não intuitiva.
+
+## História
+
+Existe um projeto central do qual todos dependem.
+Que ótimo candidato para InnerSource!
+Infelizmente, o projeto cresceu organicamente, com várias contribuições e adições feitas de forma descuidada.
+Agora é uma massa espessa e confusa de código que ninguém entende e todos têm medo de tocar.
+É claramente necessário uma revisão completa (por exemplo, refatoração, testes, documentação, etc.), mas mesmo que todos precisem e queiram que esse trabalho aconteça, ninguém tira tempo para fazê-lo.
+
+## Contexto
+
+- Muitas equipes precisam do projeto.
+- O projeto tem uma dívida técnica significativa.
+- Adoção e iteração lentas no projeto.
+- Não há um proprietário ou mantenedor que assuma a responsabilidade pelo ecossistema de projeto e contribuição como um todo.
+
+## Forças
+
+- Cada equipe contribuinte está ocupada e, portanto, prioriza o trabalho que resulta em um retorno imediato para si mesmos.
+- Conforme o projeto cresce, a tendência natural é que ele se torne mais difícil de usar e modificar.
+
+## Solução
+
+Forme uma equipe central cujo trabalho é manter este projeto em um estado para que outros possam facilmente aderir e contribuir para ele.
+Esta equipe central faz o trabalho necessário para um ecossistema de uso e contribuição saudável.
+Este trabalho crítico tende a não ser priorizado como uma contribuição.
+Categorias desse tipo de trabalho incluem comunicação, ambiente local e infraestrutura de DevOps.
+
+Aqui estão alguns exemplos específicos:
+
+- Bugs em produção
+- Documentação
+- Tutoriais e exemplos para integração de novos colaboradores
+- Testes automatizados
+- Integração e entrega contínua (CI/CD)
+- Ambiente local de desenvolvimento
+- Modularização
+- Versionamento
+- Monitoramento
+- Desbravamento de novas classes/categorias de funcionalidades.
+
+Cada um desses itens é muito importante para um ecossistema de produto saudável, mas é improvável que seja priorizado como uma contribuição.
+
+A equipe central pode ser composta por um pequeno número de pessoas em tempo integral ou parcial.
+A escolha depende da quantidade de trabalho necessário, da disponibilidade de recursos e da cultura da organização.
+A consideração mais importante é formar a equipe de forma que permita que a organização a capacite e a responsabilize da mesma maneira que qualquer outra equipe.
+
+Due to their central role, core team members should nearly always fill the role of **Trusted Committers** as well (for more on that concept see [Learning Path][tc-learning-path] and [Pattern][tc-pattern]).
+While the Trusted Committer role focuses mostly on facilitating others' contribution and use of the project, a core team member regularly contributes to the project as well.
+The core team doesn't have its own business agenda that determines its contributions.
+They decide what to work on based on what will help others most to use and contribute to the project.
```

Comment:
@jrcosta this block here is still in English. Can this be deleted?

Reply:
That's right @spier , we had forgotten to remove this already translated section. I just removed it."